### ISSUE TITLE
SA1413: Single-line enum considered multi-line

### ISSUE THREAD

**Issue by koromorix** [DotNetAnalyzers/StyleCopAnalyzers#2467]
**Code**:

```cs
enum E { A, B }
```

**Expected**: No error. (Assuming SA1502 is disabled)
**Actual**: `Program.cs(4,14): warning SA1413: Use trailing comma in multi-line initializers`
